es-vue-base/src/lib-components/EsZipCodeForm.vue

```diff
@@ -24,6 +24,7 @@
                 <es-form-input
                     :id="inputId"
                     v-model="zipCode"
+                    :autofocus="autofocus"
```

Comment:
Noting the Github warning below. Using autofocus [causes screen readers to "teleport"](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus) to the zip code form on page load, which is an accessibility failure. I confirmed this behavior with VoiceOver. This issue actually affects our homepage as well due to it being used there, and we probably need to have a broader conversation about it. @ericdouglaspratt what do you think?

Reply:
Problem solved by using the html element method "focus" which allows me to pass in the prop "preventScroll: true"

Reply:
This unfortunately doesn't solve the problem. With these changes, when you navigate to the page, the keyboard focus changes to the zip code form. This is a problem for people with vision impairment who rely on screen readers, because instead of starting at the top of the page, the screen reader will "teleport" to the zip code form.